assembl/static2/yarn.lock

```diff
@@ -8100,6 +8156,11 @@ moment-duration-format@^1.3.0:
   resolved "https://registry.yarnpkg.com/moment-duration-format/-/moment-duration-format-1.3.0.tgz#541771b5f87a049cc65540475d3ad966737d6908"
   integrity sha1-VBdxtfh6BJzGVUBHXTrZZnN9aQg=
 
+moment@>=1.6.0:
+  version "2.24.0"
+  resolved "https://registry.yarnpkg.com/moment/-/moment-2.24.0.tgz#0d055d53f5052aa653c9f6eb68bb5d12bf5c2b5b"
+  integrity sha512-bV7f+6l2QigeBBZSM/6yTNq4P2fNpSWj/0e7jQcy87A8e7o2nAfP/34/2ky5Vw4B9S446EtIhodAzkFCcR4dQg==
+
```

Comment:
Please keep only one moment version, keep the latest and merge manually `moment@>=1.6.0, moment@^2.20.1, moment@^2.22.0`

Reply:
Not sure how to do this properly tbh, I don't understand why I always corrupt my yarn.lock 😞

Reply:
Not your fault, it's the way `yarn add` works, it will get the latest version of all dependencies, so we end up having multiple versions of some packages.